src/aliceVision/multiview/essentialF10Solver.hpp

```diff
@@ -0,0 +1,235 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2018 AliceVision contributors.
+// Copyright (c) 2015 Jan Heller <hellej1@cmp.felk.cvut.cz>, Zuzana Kukelova <zukuke@microsoft.com>
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <aliceVision/numeric/numeric.hpp>
+
+#include <Eigen/Dense>
+
+#include <cmath>
+#include <limits>
+#include <vector>
+
+namespace aliceVision {
+
+using Mat21 = Eigen::Matrix<double, 2, 1>;
+
+template <typename Scalar>
+bool isZero(Scalar val)
```

Comment:
this seems not used, `Matrix.isZero()` is used instead, so I guess it can be removed